utils/s2n_safety.h

```diff
@@ -27,8 +27,8 @@
 /* Check memcpy's return, if it's not right (very unlikely!) bail, set an error
  * err and return -1;
  */
-#define memcpy_check( d, s, n )     do { notnull_check( (d) ); if ( memcpy( (d), (s), (n)) != (d) ) { S2N_ERROR(S2N_ERR_MEMCPY); } } while(0)
```

Comment:
Are S2N_ERR_MEMCPY and S2N_ERR_MEMSET ever used now after this change? should they be removed from the list of potential errors?